src/modules/luksbootkeyfile/LuksBootKeyFileJob.cpp

```diff
@@ -297,8 +323,8 @@ LuksBootKeyFileJob::exec()
 
         if ( !setupLuks( d ) )
             return Calamares::JobResult::error(
-                tr( "Encrypted rootfs setup error" ),
```

Comment:
I know you didn't write this part, but I wonder if this should not throw an error?  It seems like in most cases we would not want a failure here to cause the install to fail.

Reply:
I agree, whether or not the keyfile creation is successful or not, installation should continue to prevent an unbootable system.